### PR TITLE
Alt nav two

### PR DESCRIPTION
**Pull request by dkuntze**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Test URLs:
- Before: 
  - https://main--shredit--stericycle.aem.page/en-us/marketing/new-paper-shredding-services
- After: 
  - https://alt-nav-two--shredit--stericycle.aem.page/en-us/marketing/new-paper-shredding-services

### CODE REVIEW

**Comment by dkuntze** [comment]
image

**Comment by dkuntze** [comment]
image

**Comment by jindaliiita** [comment]
@dkuntze Somehow for me, header is not showing up

Screenshot 2024-09-26 at 2 27 14 PM

File: blocks/header/header.js [hunk 1]
```diff
@@ -718,4 +726,26 @@ export default async function decorate(block) {
   navWrapper.append(nav);
   block.append(navWrapper);
   block.parentElement.classList.add('appear');
+
+  // variant for the alt-two header
+  if (block.querySelector('.alt-two')) {
+    const sectionSubNav = block.querySelector('.nav-sections > div.default-content-wrapper > ul');
+    if (sectionSubNav) {
+      const mobileClone = sectionSubNav.cloneNode(true);
```
[the remark this answers is not in the document]
Comment (dkuntze) [reply blocks/header/header.js:734]: This nav has a separate background color that spans the whole width, not just the nav area

File: scripts/delayed.js [hunk 2]
```diff
@@ -105,3 +105,26 @@ if (document.querySelector('body.resource-center')
         || document.querySelector('meta[name="media-type"]')?.getAttribute('content') === 'Info Sheets')) {
   await loadYMAL();
 }
+
+const observer = new IntersectionObserver((entries) => {
```
[the remark this answers is not in the document]
Comment (dkuntze) [reply scripts/delayed.js]: moved

File: blocks/header/header.js [hunk 3]
```diff
@@ -718,4 +726,26 @@ export default async function decorate(block) {
   navWrapper.append(nav);
   block.append(navWrapper);
   block.parentElement.classList.add('appear');
+
+  // variant for the alt-two header
+  if (block.querySelector('.alt-two')) {
```
[the remark this answers is not in the document]
Comment (dkuntze) [reply blocks/header/header.js]: made it its own function

File: blocks/header/header.js [hunk 4]
```diff
@@ -537,14 +537,22 @@ export default async function decorate(block) {
     navTools.querySelectorAll(':scope .default-content-wrapper > ul > li').forEach((navTool) => {
       if (navTool.querySelector('a')) {
         if (navTool.querySelector('a').getAttribute('title').startsWith('Customer Service')
-            || navTool.querySelector('a').getAttribute('title').startsWith('Sales')) {
```
[the remark this answers is not in the document]
Comment (dkuntze) [reply blocks/header/header.js:540]: I'm not thrilled by it either, but there are limited options

**Review by MarquiseRosier — APPROVED** [review]

**Pull request merged**